test/mocks/ssl/mocks.h

```diff
@@ -99,6 +99,12 @@ class MockClientContextConfig : public ClientContextConfig {
   MOCK_METHOD(bool, allowRenegotiation, (), (const));
   MOCK_METHOD(size_t, maxSessionKeys, (), (const));
   MOCK_METHOD(const std::string&, signingAlgorithmsForTest, (), (const));
+
+  Ssl::HandshakerCapabilities capabilities_;
+  std::string sni_{"default_sni"};
```

Comment:
`default_sni.example.com` would be more correct, TLS has weird rules about single-label names